company_website/templates/main_page_partials/main_screen.haml

```diff
@@ -1,8 +1,13 @@
 {% load static %}
+{% load variable_tag %}
 
 .container-fluid.px-0.main-screen{:id => "home"}
     #content.jumbo.d-flex
+        {% if request.user_agent.browser.family|is_webp_supported %}
+        %img.background-img.lazyload{:alt => "Code Poets", :data-src => "{% static 'main_page/images/main_screen/slide.webp' %}", :src => "{% static 'main_page/images/main_screen/slide_placeholder.webp' %}", :data-srcset => "{% static 'main_page/images/main_screen/slide_500x1248.webp' %} 500w, {% static 'main_page/images/main_screen/slide_850x1248.webp' %} 850w, {% static 'main_page/images/main_screen/slide.webp' %} 1200w", :srcset => "{% static 'main_page/images/main_screen/slide_500x1248_placeholder.webp' %} 500w, {% static 'main_page/images/main_screen/slide_850x1248_placeholder.webp' %} 850w, {% static 'main_page/images/main_screen/slide_placeholder.webp' %} 1200w"}
+        {% else %}
```

Comment:
Why here you haven't used same mechanism as on other pages? We shouldn't duplicate code, please fix this :)

Reply:
Done